db/src/users.rs

```diff
@@ -6,6 +6,7 @@ use uuid::Uuid;
 pub struct User {
     pub user_id: Uuid,
     pub user_name: String,
+    pub password_hash: Option<String>,
```

Comment:
This is an `Option` since not all the existing users in your local DB will have a password_hash

But all Users we create from the web side are expected to have password_hashes!

Eventually we will likely do some sort of 'backfill' to get password hashes into all our DBs so that we can make this column non-null (and then remove this `Option`) here!